quickstarts/microsoft.digitaltwins/digitaltwins-with-function-private-link/README.md

```diff
@@ -0,0 +1,55 @@
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.digitaltwins/digitaltwins-with-function-private-link/CredScanResult.svg)
+
+![Bicep Version](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.digitaltwins/digitaltwins-with-function-private-link/BicepVersion.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.digitaltwins%2Fdigitaltwins-with-function-private-link%2Fazuredeploy.json)
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.digitaltwins%2Fdigitaltwins-with-function-private-link%2Fazuredeploy.json)
+
+This template creates an Azure Digital Twins service configured with a Virtual Network connected Azure Function that can communicate through a Private Link Endpoint to Digital Twins. It also creates a Private DNS Zone to allow seamless hostname resolution of the Digital Twins Endpoint from the Virtual Network to the Private Endpoint internal subnet IP address. The hostname is stored as a setting to the Azure Function with name 'ADT_ENDPOINT'.
+
+Tags: `Azure Digital Twins, Private Link, Azure Functions, Virtual Network, Managed Identities, MSI, Role-based Access Control, RBAC`
+
+## Sample overview and deployed resources
+
+This sample template creates an Azure Digital Twins instance, Private Link service, Private DNS Zone, an Azure Function with a system-assigned identity and a role-assignment for the Azure Function to access Digital Twins as Azure Digital Twins Data Owner.
+
+The following resources are deployed as part of the solution:
+
+## Networking
+
+- Virtual Network, with subnets for the Azure Function and and for Private Link service.
+- Private Link service to make the Azure Digital Twins instance accessible to the Azure Function through a private endpoint.
+
+## Azure Function
+
+- An Azure Function configured with a Server Farm and a Network Profile that connects it to the Virtual Network
+
+## Azure Digital Twins
+
+- An Azure Digital Twins instance configured to not accept connections from the public network by setting the `PublicNetworkAccess` policy to `Disabled`.
+
+The following diagram illustrates the components of this sample.
+
+![Architecture diagram showing traffic going from the Azure Function via Private Link service to Digital Twins.](images/diagram.png "Diagram highlighting the architecture of the deployed resources")
```

Comment:
If this diagram is supposed to illustrate the components listed above, you might want to make the text common between them (the list says "Azure Function" while the diagram says "Function Apps;" the list mentions "Azure Digital Twins" while the diagram gives the endpoint name as "ADT_ENDPOINT"). This isn't a big deal but it could be extra documentation clarity.